apps/examples/fs_helper/Kconfig

```diff
@@ -24,6 +24,14 @@ config ROMFS_TEST
 	---help---
 		Enable "romfs_test" application
 
+config FS_RW_TEST
+	bool "fs rw test"
+	depends on TASH
+	default n
```

Comment:
it only depends on tash? I think it needs more features for example smartfs ...

Reply:
add deoebds on smartfs